Mathlib/Order/Interval/Set/Basic.lean

```diff
@@ -38,37 +38,6 @@ section Preorder
 
 variable [Preorder α] {a a₁ a₂ b b₁ b₂ c x : α}
 
-/-- `Ioo a b` is the left-open right-open interval $(a, b)$. -/
-def Ioo (a b : α) :=
-  { x | a < x ∧ x < b }
-
-/-- `Ico a b` is the left-closed right-open interval $[a, b)$. -/
-def Ico (a b : α) :=
-  { x | a ≤ x ∧ x < b }
-
-/-- `Iio b` is the left-infinite right-open interval $(-∞, b)$. -/
-def Iio (b : α) :=
-  { x | x < b }
-
-/-- `Icc a b` is the left-closed right-closed interval $[a, b]$. -/
-def Icc (a b : α) :=
-  { x | a ≤ x ∧ x ≤ b }
-
-/-- `Iic b` is the left-infinite right-closed interval $(-∞, b]$. -/
-def Iic (b : α) :=
-  { x | x ≤ b }
-
-/-- `Ioc a b` is the left-open right-closed interval $(a, b]$. -/
-def Ioc (a b : α) :=
-  { x | a < x ∧ x ≤ b }
-
-/-- `Ici a` is the left-closed right-infinite interval $[a, ∞)$. -/
-def Ici (a : α) :=
-  { x | a ≤ x }
-
-/-- `Ioi a` is the left-open right-infinite interval $(a, ∞)$. -/
-def Ioi (a : α) :=
-  { x | a < x }
 
 theorem Ioo_def (a b : α) : { x | a < x ∧ x < b } = Ioo a b :=
```

Comment:
Should these results also move into `Defs`?

Reply:
I have no preference here. Moved.